lms/static/scripts/frontend_apps/components/LMSFilePicker.js

```diff
@@ -263,14 +321,28 @@ export default function LMSFilePicker({
       {warningOrError}
 
       {['reloading', 'fetched'].includes(dialogState.state) && (
-        <FileList
-          files={dialogState.files ?? []}
-          isLoading={dialogState.state === 'reloading'}
-          selectedFile={selectedFile}
-          onUseFile={onSelectFile}
-          onSelectFile={selectFile}
-          noFilesMessage={<NoFilesMessage href={missingFilesHelpLink} />}
-        />
+        <Fragment>
```

Comment:
```suggestion
        <>
```

Reply:
See related comment, but this project doesn't support this syntax (yet).